polaris.shopify.com/src/components/PropsTable/PropsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { PropsForComponent } from "../../types";
+import StatusBadge from "../StatusBadge";
+import styles from "./PropsTable.module.scss";
+
+interface Props {
+  props: PropsForComponent;
+}
+
+function PropsTable({ props: { interfaceName, props } }: Props) {
+  const feedbackTitle = "[polaris.shopify.com] Props table feedback";
+  const feedbackUrl = `https://github.com/shopify/polaris/issues/new?title=${encodeURIComponent(
+    feedbackTitle
+  )}&amp;labels=polaris.shopify.com`;
+
+  return (
+    <div className={styles.PropsTable}>
+      <h2 id="properties">Props</h2>
+      <p>
+        This feature is a work in progress. Please{" "}
+        <a href={feedbackUrl}>share your feedback</a>.
+      </p>
```

Comment:
I liked this framing for new features from GitHub:

> Want to help make code search better? Provide [feedback here!](https://github.com/github/feedback/discussions/categories/code-search-and-navigation-feedback)

Reply:
A `new` or `alpha` badge or something might also be nice to show that things are still being worked on.

Reply:
I like that! Changed it to:

> Want to help make this feature better? Please [share your feedback](https://github.com/shopify/polaris/issues/new?title=%5Bpolaris.shopify.com%5D%20Props%20table%20feedback&amp;labels=polaris.shopify.com).